src/de/de.json

```diff
@@ -7,51 +7,90 @@
     "language": "German",
     "languageNative": "Deutsch",
     "source": [
-      "https://en.wikipedia.org/wiki/German_orthography#Special_characters"
+      "https://en.wikipedia.org/wiki/German_orthography#Special_characters",
+      "https://en.wikipedia.org/wiki/Capital_%E1%BA%9E"
     ]
   },
   "data": {
     // mapping information
     "ü": {
       "mapping": {
         "base": "u",
-        "decompose": "ue"
+        "decompose": {
+          "titleCase": "Ue",
+          "upperCase": "UE",
+          "lowerCase": "ue"
+        }
       }
     },
     "Ü": {
       "mapping": {
         "base": "U",
-        "decompose": "UE"
+        "decompose": {
+          "titleCase": "Ue",
+          "upperCase": "UE",
+          "lowerCase": "ue"
+        }
       }
     },
     "ö": {
       "mapping": {
         "base": "o",
-        "decompose": "oe"
+        "decompose": {
+          "titleCase": "Oe",
+          "upperCase": "OE",
+          "lowerCase": "oe"
+        }
       }
     },
     "Ö": {
       "mapping": {
         "base": "O",
-        "decompose": "OE"
+        "decompose": {
+          "titleCase": "Oe",
+          "upperCase": "OE",
+          "lowerCase": "oe"
+        }
       }
     },
     "ä": {
       "mapping": {
         "base": "a",
-        "decompose": "ae"
+        "decompose": {
+          "titleCase": "Ae",
+          "upperCase": "AE",
+          "lowerCase": "ae"
+        }
       }
     },
     "Ä": {
       "mapping": {
         "base": "A",
-        "decompose": "AE"
+        "decompose": {
+          "titleCase": "Ae",
+          "upperCase": "AE",
+          "lowerCase": "ae"
+        }
       }
     },
     "ß": {
       "mapping": {
         "base": "ß",
-        "decompose": "ss"
+        "decompose": {
+          "titleCase": "SS",
+          "upperCase": "SS",
+          "lowerCase": "ss"
+        }
+      }
+    },
+    "ẞ": {
+      "mapping": {
+        "base": "ß",
+        "decompose": {
+          "titleCase": "ss",
```

Comment:
Why is great sharp-s ss in title case? Wouldn't it be SS?

Reply:
Oh, you mentioned that `ß` would never be at the start of a word, so in title case, it should be in lower case.

Reply:
You're right. But a diacritic `ü` may also occur in the middle of a title case word, then it also must be `ue` and not `Ue`. So from my point of view the title case property is really only meant for if the diacritic is the first character in a word. In that case `Ss` would be correct I guess, even when it never occurs.

Reply:
Ok